getting-started/deployment/aws-ec2.mdx

```diff
@@ -87,11 +87,100 @@ After making changes, restart the Docker Compose containers:
 sudo docker compose down && sudo docker compose up -d
 ```
 
-## TLS
+## Enabling TLS (HTTPS) access with a load balancer
 
-For production usage, it’s recommended to enable TLS (HTTPS). You can achieve this by adding an EC2 Load Balancer to serve traffic to your EC2 instance.
+For production usage, it's recommended to enable TLS (HTTPS). Without TLS, cloud templates may not load properly when using the Safari browser.
 
```

Comment:
```suggestion
For production usage, it's recommended to enable TLS (HTTPS). In addition to the security aspect, this will also ensure cloud templates load properly on different browsers.
```

Reply:
Incorporated the suggestion